views/works/show.haml

```diff
@@ -64,7 +64,7 @@
                       %i.fa{class: "fa-#{social['icon']}"}
                       %span.hidden
                         = social['icon']
-
+-#
       - if @works[:data].length > 0
```

Comment:
I think this will not work with your code in `app.rb`. You might want to add the following in `app.rb`:

```
@works = {}
```